op-challenger/challenger/dispatch.go

```diff
@@ -0,0 +1,39 @@
+package challenger
+
+// dispatch runs the dispatch service for the challenger.
+// This function spawns a goroutine.
+func (c *Challenger) dispatch() {
+	c.wg.Add(1)
+	go func() {
```

Comment:
often we have a named function for the event loop to avoid a level of indentation